.ci/create-changes-html.sh

```diff
@@ -19,7 +19,7 @@ echo '<script>hljs.highlightAll();</script>' >> CHANGES.html
 cat >> CHANGES.html << EOF
 <script>
 document.addEventListener('DOMContentLoaded', () => {
-const baseDocURL = 'https://sagemath.netlify.app'
+const baseDocURL = 'https://preview-release--sagemath.netlify.app'
```

Comment:
I don't understand the naming changes in this PR

Reply:
We are uploading docs for a PR to, for example, https://preview-38220--sagemath.netlify.app, which is an "alias" of a non-human-readable url. 

On the other hand, docs for a release is uploaded to the production site `https://sagemath.netlify.app`, whose alias is, after this PR,  `https://preview-release--sagemath.netlify.app`. Before this PR, the alias was `https://preview-sagemath.netlify.app` but this alias url was not used.

Hence the naming change is just to make the url of the doc preview of the release look similar to the url for doc preview of a PR. But this change is not strictly necessary.

Do you prefer not to use the alias and keep using the shorter url?

Reply:
I just checked that https://preview--sagemath.netlify.app does not exist...

This means that there is no alias for the production site. If so, the change on line 22 will not work...

Reply:
There is a problem in using `--prod` together with `--alias`. That doesn't work as I described above. See https://answers.netlify.com/t/netlify-cli-deploy-with-alias-not-publishing/108047

Hence https://github.com/sagemath/sage/pull/38220/commits/0fc6bbe46af30d381a3d8c1b463c717b9d9d1a08, I removed `--prod`.

Thus we will only have: https://preview-release--sagemath.netlify.app for doc preview for releases.

Reply:
Then the production site https://sagemath.netlify.app remains unused. We may use it for other purpose. Perhaps we may deploy livedoc for stable versions there. Something to consider later.

Reply:
Or I can do it here if we agree on how to use `https://sagemath.netlify.app/`.

Reply:
Yes, let's try to figure it out, keeping also the following things in mind:
- this wishlist item: #37384;
- a possible future change in how the "develop" branch is operated, in which not only a beta or stable release is pushed to "develop";
- that with #36730, wheels of the livedoc are pushed to PyPI on beta or stable releases.

Some general thoughts: I think the word "preview" should only appear in what we build for PRs. Documentation that is built for a release tag (beta or stable), and documentation for the latest commit on the "develop" branch is not a "preview".

Reply:
I think the doc built for a release tag is still a "preview", since it is not an "official" deployment of the doc. Currently The deployment to https://doc.sagemath.org is of course official. The deployment to currently https://deploy-livedoc--sagemath.netlify.app (or https://livedoc--sagemath.netlify.app in future) might be called "official".

Reply:
But I am okay with `release--sagemath.netlify.app` instead of `preview-release--sagemath.netlift.app`. If we plan to add `release-version--sagemath.netlify.app`, then prefix `preview` would look redundant.